apps/stack/stacks/TransformStack.ts

```diff
@@ -0,0 +1,32 @@
+import {
+  use,
+  Config,
+  type StackContext,
+} from "sst/constructs";
+import { ExtractStack } from "./ExtractStack";
+
+export function TransformStack({ stack }: StackContext) {
+
+  const { ExtractBus } = use(ExtractStack);
+  const TRANSFORM_DB_URL = new Config.Secret(stack, "TRANSFORM_DB_URL");
+  const TRANSFORM_DB_AUTH_TOKEN = new Config.Secret(stack, "TRANSFORM_DB_AUTH_TOKEN");
+
+  ExtractBus.addRules(stack, {
+    "transformRepository": {
+      pattern: {
+        source: ["extract"],
+        detailType: ["repository"]
+      },
+      targets: {
+        transformRepository: {
+          function: {
+            bind: [TRANSFORM_DB_URL, TRANSFORM_DB_AUTH_TOKEN],
+            handler: "src/transform/transform-repository.eventHandler",
+          }
+        }
+      },
+      
+    }
+  });
```

Comment:
Had to create a new rule in ExtractBus for transform because reusing existing creates a circular dependency between ExtractStack and TransformStack.

Additionally, ExtractBus function defaults are applied to new rules as well (bind DATABASE and others..)